plane/source/docs/apms-failsafe-function.rst

```diff
@@ -100,21 +100,21 @@ Meaning that when flying, our throttle values will range between 1100 -
 #. Turn off the transmitter. You should see the value drop
    significantly. This will be the PWM value relayed to the autopilot in
    the event RC link was lost during flight.
-#. Make sure THR_FS_VALUE is an adequate number to trigger the
+#. Make sure :ref:`THR_FS_VALUE<THR_FS_VALUE>` is an adequate number to trigger the
    failsafe function on the autopilot.
-#. Make sure FS_SHORT_ACTN and FS_LONG_ACTN are both enabled (set to
-   1).
+#. Make sure :ref:`FS_SHORT_ACTN<FS_SHORT_ACTN>` and :ref:`FS_LONG_ACTN<FS_LONG_ACTN>` are both enabled (set to
```

Comment:
They don't have to both be enabled, you can actually disable either of them safely and the other still works. IE I fly with just a long failsafe enabled.

Reply:
fixed